projects/addon-doc/src/components/example/example-capitalize.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {capitalize} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+@Pipe({name: 'tuiDocExampleCapitalize'})
+export class TuiDocExampleCapitalizePipe implements PipeTransform {
+    transform(content: PolymorpheusContent): PolymorpheusContent {
+        return typeof content === 'string' ? capitalize(content) : content;
```

Comment:
oh, according to screenshot tests something goes wrong

Reply:
we can inject [TitleCasePipe](https://angular.io/api/common/TitleCasePipe) and use its transform.

What do you think ?

Reply:
If I try inject `TitleCasePipe` I have

<img width="612" alt="image" src="https://user-images.githubusercontent.com/12021443/160388959-57580b18-702d-49c6-9614-d3966647105b.png">

Reply:
also I applied the wrong logic, because TitleCasePipe or `capitalize` util from taiga do
`Capitalizes the first letter of each word and transforms`

Reply:
fixed

Reply:
> If I try inject TitleCasePipe I have

See example how to get this DI-entity
https://github.com/Tinkoff/taiga-ui/blob/main/projects/kit/pipes/field-error/field-error-content-pipe.ts#L18

Reply:
yes, no problem, but this pipe is not suitable for this issue

Reply:
because we need to use capitalize first letter

Reply:
my mistake